Neos.Neos/Documentation/References/PropertyEditorReference.rst

```diff
@@ -400,6 +400,35 @@ Options Reference:
 ``threshold`` (number)
 	Minimum amount of characters which trigger a search. Default is set to 2.
 
+``createNew`` (array)
+    If you are using the rewritten Neos Ui, it is also possible to create new selectable nodes directly from the reference editor.
```

Comment:
Saying "rewritten Neos UI" doesn't age well, it's like saying "New School of XY". It's new for a while, but not for too long. I would assume that people using Neos 3.3 will also use the new UI since it's enabled by default.
If you keep it this way, I still suggest to write UI in uppercase.

Reply:
I'd also say `s/rewritten Neos Ui/current Neos user insterface/`. People don't need to know that there existed some other UI, besides the current one...